config/shared_config.go

```diff
@@ -31,7 +31,7 @@ const (
 
 	// Prefix for services section. It is referenced in profile via the services
 	// parameter to configure clients for service-specific parameters.
-	servicesPrefix = `services`
+	servicesPrefix = `services `
```

Comment:
is this extra space important? if so, why?

Reply:
Yeah, it matches the way we define all the other section headers because the profile name is stored as declared (e.g. `Sections.Get("services foo")`).